certbot/certbot/_internal/plugins/disco.py

```diff
@@ -44,15 +48,15 @@ class PluginEntryPoint:
     # this object is mutable, don't allow it to be hashed!
     __hash__ = None  # type: ignore
 
-    def __init__(self, entry_point, with_prefix=False):
+    def __init__(self, entry_point: pkg_resources.EntryPoint, with_prefix=False):
         self.name = self.entry_point_to_plugin_name(entry_point, with_prefix)
-        self.plugin_cls = entry_point.load()
+        self.plugin_cls: Any = entry_point.load()
         self.entry_point = entry_point
-        self.warning_message = None
-        self._initialized = None
-        self._prepared = None
+        self.warning_message: Optional[str] = None
+        self._initialized: Optional[Any] = None
```

Comment:
I have a similar comment here as I do about `self.plugin_cls` except here the type should be `interfaces.IPlugin` and `mypy` complains about the number of parameters on the `prepare` call.

Reply:
See above.